### DISCUSSION TITLE
Astroid Framework @ 3.1.6 but Astroid Plugin @ 3.1.0 -> how do I update?

### DISCUSSION THREAD

**Discussion by schae09** [templaza/astroid-framework#837] [Q&A]
The Pre-Update Check for Joomla ‎5.2.0 shows the error message that Astroid has update requirements. When investigating closer I found out that Astroid Framework @ 3.1.6 but Astroid Plugin is just @ 3.1.0 and Astroid Clear Cache and Astroid Menu are @ 3.1.1

I downloaded the Framework 3.1.6 here from GitHub and installed it, but there was no change in the version numbers. Also clearing the cache and reinstalling again were not successful. What am I doing wrong?

**Comment by sonvnn** [comment]
Please update to version 3.1.7

**Reply by schae09** [reply]
Thanks a lot. I did and it works ;-)
